### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by liubin**
Signed-off-by: bin liu liubin0329@gmail.com

### CODE REVIEW

**Comment by hqhq** [comment]
Hi liubin,

vendor codes should be changed in it's own project.

File: vendor/src/github.com/godbus/dbus/conn.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func (conn *Conn) Close() error {
```
Comment (unclejack) [inline vendor/src/github.com/godbus/dbus/conn.go]: Changing this file like this isn't allowed. You need to send a pull request against that `github.com/godbus/dbus` repository first and update the vendored version.

**Comment by icecrime** [comment]
@liubin As explained previously by other reviews, could you please amend your commit to keep the change to `pkg/term/winconsole/term_emulator_test.go` only?

File: vendor/src/github.com/godbus/dbus/conn.go (hunk 1 above)
Comment (liubin) [inline vendor/src/github.com/godbus/dbus/conn.go]: OK,thanks.

**Comment by thaJeztah** [comment]
Thanks! LGTM

/ping @icecrime

**Comment by duglin** [comment]
LGTM

**Pull request merged**